### ISSUE TITLE
Add Smoothness to the way Tagging schema, not just for Tracks

### ISSUE THREAD

**Issue by jjiglesiasg** [openstreetmap/id-tagging-schema#1070]
### OSM Tag(s)

smoothness=

### How would you like this tag to see supported?

As a Preset

### Label

smoothness

### Link to OSM Wiki page

https://wiki.openstreetmap.org/wiki/Key:smoothness

### Status of the Tag

Approved

### Usage of the tag

4188217

### Replaces other Tag?

No

### Regional Tag?

global

### Further Information

Add the Smoothness tag as data to be filled for any kind of ways, not just only for tracks. The classification of a way is NOT dependent of the Smoothness or Ruggedness of such way. So adding that value make sense in all kind of roads

**Comment by tyrasd** [comment]
agree, though I'd probably still exclude it from the highest grade highway types, i.e. motorway (maybe also trunk), as for these roads a good pavement condition is kind of implied. For the remaining road types, probably the best would be to include the Smoothness tag as an optional field. This would still allow for countries/regions where unpaved minor roads are common to override these presets by a regional one.
